napari/_qt/dialogs/qt_notification.py

```diff
@@ -405,9 +405,11 @@ def _debug_tb(tb):
     QApplication.processEvents()
     QApplication.processEvents()
     with event_hook_removed():
-        print("Entering debugger. Type 'q' to return to napari.\n")
+        print(  # noqa: T201
```

Comment:
`loggin.info`?

Reply:
Not sure how to deals with these cause I'm not familiar with this module. Isn't this print here on purpose so the traceback can be shown to the user?